### ISSUE TITLE
Error with hanging indent for single assignment let/const/var objects

### ISSUE THREAD

**Issue by andyearnshaw** [eslint/eslint#8616]
**Tell us about your environment**

* **ESLint Version:** v3.19.0
* **Node Version:** v7.8.0
* **npm Version:** v4.2.0

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint

**Please show your full configuration:**

```yaml
root: true

parser: 'babel-eslint'
parserOptions:
  sourceType: module

env:
  browser: true
  es6: true
  jest: true

rules:
  indent:
    - error
    - 4
    - SwitchCase: 1
  one-var: 'off'
  one-var-declaration-per-line:
    - error
    - initializations
```

**What did you do? Please include the actual source code causing the issue.**

```js
let obj = {
        a: 1,
        b: 2,
    };
```

**What did you expect to happen?**

I expected this to be valid.  Mainly because this is:

```js
let a,
    obj = {
        b: 1,
        c: 2,
    };
```

**What actually happened? Please include the actual, raw output from ESLint.**

```
test.js|2 col 9 error| Expected indentation of 4 spaces but found 8. (indent)
test.js|3 col 9 error| Expected indentation of 4 spaces but found 8. (indent)
test.js|4 col 5 error| Expected indentation of 0 spaces but found 4. (indent)
|| 
|| 3 problems
```

I prefer this kind of indent because I feel it's more consistent.  Declaring only one variable should not change my indent style.

**Comment by not-an-aardvark** [comment]
Closing as a duplicate of https://github.com/eslint/eslint/issues/3352

**Comment by andyearnshaw** [comment]
Thanks, I searched but couldn't find it.
